Translations/pt_BR.po

```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
 msgstr "Demônio"
 
 #: Source/monstdat.cpp:91
-#, fuzzy
-#| msgid "Undead Balrog"
 msgctxt "monster"
 msgid "Undead Balrog"
-msgstr "Morto-vivo"
+msgstr "Balrog Morto-vivo"
 
 #: Source/monstdat.cpp:92
-#, fuzzy
-#| msgid "Incinerator"
 msgctxt "monster"
 msgid "Incinerator"
 msgstr "Incinerador"
 
 #: Source/monstdat.cpp:93
-#, fuzzy
-#| msgid "Flame Lord"
 msgctxt "monster"
 msgid "Flame Lord"
-msgstr "da chama"
+msgstr "Senhor da Chama"
 
 #: Source/monstdat.cpp:94
-#, fuzzy
-#| msgid "Doom Fire"
 msgctxt "monster"
 msgid "Doom Fire"
-msgstr "Fogo "
+msgstr "Fogo da Perdição"
 
 #: Source/monstdat.cpp:95
-#, fuzzy
-#| msgid "Hell Burner"
 msgctxt "monster"
 msgid "Hell Burner"
-msgstr "Descer para o inferno"
+msgstr "Quimador do Inferno"
 
 #: Source/monstdat.cpp:96
-#, fuzzy
-#| msgid "Red Storm"
 msgctxt "monster"
 msgid "Red Storm"
-msgstr "Ápice da tempestade"
+msgstr "Tempestade Vermelha"
 
 #: Source/monstdat.cpp:97
-#, fuzzy
-#| msgid "Storm Rider"
 msgctxt "monster"
 msgid "Storm Rider"
-msgstr "Ápice da tempestade"
+msgstr "Piloto da Tempestade"
```

Comment:
This looks terrible... I don't have a suggestion though.

Reply:
Hahah, yeah. I am trying to think about names following a similar pattern but sometimes is really hard.